src/App.jsx

```diff
@@ -587,7 +570,7 @@ const LandingPage = ({ onLaunchDashboard }) => {
             {/* Call to Action */}
             <div className="bg-gradient-to-br from-blue-600 to-purple-700 rounded-xl p-8 text-center shadow-xl">
               <div className="bg-white bg-opacity-10 rounded-xl p-6 backdrop-blur-sm">
-                <h3 className="text-2xl font-bold mb-3 text-white">Ready to Get Started?</h3>
+                <h3 className="text-2xl font-bold mb-3 ">Ready to Get Started?</h3>
```

Comment:
We can remove this dangling white space in the className section:
```suggestion
                <h3 className="text-2xl font-bold mb-3">Ready to Get Started?</h3>
```